common/src/main/java/com/railwayteam/railways/events/ClientEvents.java

```diff
@@ -25,25 +25,25 @@ public static void onClientStarted(Minecraft mc) {
 
     public static void onClientTickStart(Minecraft mc) {
         PhantomSpriteManager.tick(mc);
-        if (DummyRailwayMarkerHandler.getInstance() == null)
-            return;
 
-        Level level = mc.level;
-        long ticks = level == null ? 1 : level.getGameTime();
-        if (ticks % 40 == 0 && previousDevCapeSetting != (previousDevCapeSetting = CRConfigs.client().useDevCape.get())) {
-            CRPackets.PACKETS.send(new ConfigureDevCapeC2SPacket(previousDevCapeSetting));
-        }
-        if (ticks % CRConfigs.client().journeymapRemoveObsoleteTicks.get() == 0) {
-            DummyRailwayMarkerHandler.getInstance().removeObsolete();
-            DummyRailwayMarkerHandler.getInstance().reloadMarkers();
-        }
+        if (DummyRailwayMarkerHandler.getInstance() != null) {
+            Level level = mc.level;
+            long ticks = level == null ? 1 : level.getGameTime();
+            if (ticks % 40 == 0 && previousDevCapeSetting != (previousDevCapeSetting = CRConfigs.client().useDevCape.get())) {
+                CRPackets.PACKETS.send(new ConfigureDevCapeC2SPacket(previousDevCapeSetting));
+            }
+            if (ticks % CRConfigs.client().journeymapRemoveObsoleteTicks.get() == 0) {
```

Comment:
Please move lines 30-34 to be outside the `DummyRailwayMarkerHandler.getInstance() != null` check